integration-tests/constants.go

```diff
@@ -0,0 +1,36 @@
+package integrationtests
+
+import (
+	"math/big"
+
+	"github.com/0xProject/0x-mesh/constants"
+)
+
+const (
+	ethereumRPCURL  = "http://localhost:8545"
+	ethereumChainID = 1337
+
+	rpcPort        = 60501
+	protocolString = "/meshsub/1.0.0"
```

Comment:
This isn't part of the public interface and I don't think there's any real reason to include it in the tests. The tests will already fail here and inside the `p2p` package if for some reason peers are not using the same protocol ID for pubsub.